addons/missionTesting/functions/fnc_updateCheckedCB.sqf

```diff
@@ -0,0 +1,48 @@
+/*
+ * Author: BadWolf
+ * Function is called by a control EH everytime that a CB is changed in the checklist.
+ *
+ * Arguments:
+ *
+ * Examples:
+ * [checkBoxControl,controlCheckedState] call potato_missionTesting_fnc_updateCheckedCB;
+ *
+ * Public: No
+ */
+
+#include "script_component.hpp"
+TRACE_1("Params",_this);
+
+params ["_control", "_checked"];
+
+private _idc = ctrlIDC _control;
+private _str = str _idc;
+private _array = _str splitString "";
+private _subsection = parseNumber (_array select 1);
+private _lineTens = parseNumber (_array select 2);
+private _lineUnits = parseNumber (_array select 3);
+private _line = (_lineTens * 10) + _lineUnits;
+private _masterChecklist = nil;
+private _checkTF = A_CHECK_TF select _checked;
+private _missionMaker = getMissionConfigValue ["author","????"];
+
+if(_missionMaker == name player) then {
+    _masterChecklist = GVAR(MissionMakerChecklistMaster);// Needs to Be changed back after testing "MissionMakerChecklistMaster"
```

Comment:
Does this need to be changed now or later?

Reply:
Comment left in error. has already been changed. Comment removed.